src/http/request_handler.rs

```diff
@@ -24,14 +25,14 @@ fn not_found() -> Result<Response<Body>> {
   Ok(Response::builder().status(StatusCode::NOT_FOUND).body(Body::empty())?)
 }
 
-fn create_request_context(req: &Request<Body>, server_ctx: &ServerContext) -> RequestContext {
+pub fn create_request_context(req: &Request<Body>, server_ctx: &ServerContext) -> RequestContext {
```

Comment:
as we have now different handlers and we have shared logic between shared handlers, it might be more suitable to have split it to 3 files: "api", "graphql", "shared". wdyt?

Reply:
@meskill with shared you mean the common methods between `api` and `graphql` right?

Reply:
yes